include/seqan3/alignment/pairwise/detail/policy_optimum_tracker.hpp

```diff
@@ -149,9 +157,11 @@ class policy_optimum_tracker
      * algorithm requires it, for example when a global alignment shall be computed.
      */
     template <typename cell_t>
-    decltype(auto) track_final_cell(cell_t && cell) noexcept
+    decltype(auto) track_final_cell(cell_t && cell, matrix_coordinate coordinate) noexcept
     {
-        m_optimal_score = (cell.optimal_score() >= m_optimal_score) ? cell.optimal_score() : m_optimal_score;
+        m_optimal_score = (cell.optimal_score() >= m_optimal_score) ?
+                          (m_optimal_coordinate = std::move(coordinate), cell.optimal_score()) :
+                           m_optimal_score;
```

Comment:
```suggestion
        auto better_score = (cell.optimal_score() >= m_optimal_score);
        m_optimal_coordinate =  better_score ? std::move(coordinate) : m_optimal_coordinate;
        m_optimal_score =  better_score ? cell.optimal_score() : m_optimal_score;
```

Would be better readable and should have the same effect? (I would expect that the optimizer would find the most efficient solution here, with both representations)